lib/xclarity_client/node_management.rb

```diff
@@ -33,5 +44,15 @@ def set_loc_led_state(uuid, state, name = 'Identify')
 
       do_put("#{Node::BASE_URI}/#{uuid}", request)
     end
+
+    private
+
+    def send_power_request(uri, requested_state = nil)
+      power_request = JSON.generate(powerState: requested_state)
+      response = do_put(uri, power_request)
+      msg = 'Power state action has been sent with request #{power_request}'
```

Comment:
I believe that string interpolation in Ruby only works with double quotes so please change this line to use double quotes.